tools/amdxdna_deps.sh

```diff
@@ -6,6 +6,11 @@
 
 SCRIPT_DIR=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
 
-apt-get install -y jq
+# if has apt-get
+if [ -x "$(command -v apt-get)" ]; then
+    apt-get install -y jq
+elif [ -x "$(command -v dnf)" ]; then
+    dnf install -y jq
+fi
```

Comment:
Seems the other deps script of xrt uses yum instead of dnf (probably for compatibility?)

So, maybe here also use yum?

Reply:
DNF has been the default package manager since EL 8. How about let's _not_ use it's legacy alias?